### DISCUSSION TITLE
Redodo BMS

### DISCUSSION THREAD

**Discussion by gottaloveit** [mr-manuel/venus-os_dbus-serialbattery#112] [New BMS requests]
Would like to see [Redodo](https://www.redodopower.com/) BMS integration.  The Home Assistant project [BMS_BLE-HA](https://github.com/patman15/BMS_BLE-HA) recently supports Redodo and seems to work well.  I'm currently using this [Redodo Battery with Bluetooth](https://www.redodopower.com/products/redodo-12v-140ah-lithium-battery-with-bluetooth)

**Comment by enzon0** [comment]
I also use this battery. Unfortunately, monitoring it via smartshunt is futile. Readings of SOC Smartshunt - BMS may vary by up to 10% per day.

https://github.com/patman15/BMS_BLE-HA
https://github.com/calledit/LiTime_BMS_bluetooth/tree/main

**Comment by gottaloveit** [comment]
I found a solution to get aggregated data of my pair of Redodo Batteries with Bluetooth BMS into Venus OS.  Since it does not use this addon to Venus OS, I posted my notes to [here](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/discussions/40)

**Comment by mr-manuel** [comment]
Will be added with https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues/313
